tests/examples/wibox/layout/stack/offset.lua

```diff
@@ -0,0 +1,15 @@
+local generic_widget = ... --DOC_HIDE
+local wibox     = require("wibox") --DOC_HIDE
+
+return --DOC_HIDE
+wibox.widget {
+    generic_widget( "first"  ),
+    generic_widget( "second" ),
+    generic_widget( "third"  ),
+    horizontal_offset = 5,
+    vertical_offset   = 5,
```

Comment:
Are these stack tests shown somewhere in the docs? If not, why the `DOC_HIDE`s? If yes, where? Shouldn't they be part of that commit?